terraform/modules/hub/prometheus.tf

```diff
@@ -276,3 +277,33 @@ resource "aws_lb_listener_rule" "prometheus_https" {
     values = ["prom-${count.index + 1}.*"]
   }
 }
+
+resource "aws_s3_bucket_policy" "access_prometheus_config" {
+  bucket = "${aws_s3_bucket.deployment_config.id}"
+
+  policy = <<POLICY
+{
+  "Id": "Policy1547564114126",
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "Stmt1547564111145",
+      "Action": [
+        "s3:GetObject"
+      ],
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::gds-${var.deployment}-config/prometheus/prometheus.yml",
```

Comment:
Also this could be `${aws_s3_bucket.deployment_config.arn}/prometheus/prometheus.yml`
Or just namespace the directory `${aws_s3_bucket.deployment_config.arn}/prometheus/*`

Reply:
also sid and id are not necessary